crates/bootstrap_srv/src/config.rs

```diff
@@ -0,0 +1,84 @@
+//! config types.
+
+/// Configuration for running a BootstrapSrv.
+#[derive(Debug)]
+pub struct Config {
+    /// Worker thread count.
+    ///
+    /// This server is currently built using blocking io and filesystem
+    /// storage. It is therefore beneficial to have more worker threads
+    /// than system cpus, since the workers will be bound on io, not
+    /// on cpu. On the other hand, increasing this will also increase
+    /// memory overhead and tempfile handle count, so we don't want to
+    /// set it too high.
+    ///
+    /// Defaults:
+    /// - `testing = 2`
+    /// - `production = 4 * cpu_count`
+    pub worker_thread_count: usize,
```

Comment:
mmm... I'm doing that as a DRY tool, since the same cleanup logic was required. Many other threads will be created by the tiny_http library itself, and who knows about any others for tracing libraries, etc. I'm not sure we need to enumerate them all.